src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -4675,4 +4675,7 @@ To remove the warning, you can use /reference instead (set the Embed Interop Typ
   <data name="ERR_TooManyUserStrings" xml:space="preserve">
     <value>Combined length of user strings used by the program exceeds allowed limit. Try to decrease use of string literals.</value>
   </data>
-</root>
\ No newline at end of file
+  <data name="ERR_PeWritingFailure" xml:space="preserve">
+    <value>An error occurred while writing the Portable Executable file.</value>
```

Comment:
Do you think that "the output file" is specific enough?  "output file" could refer to a pdb.

Maybe "target"?